src/user/user.ts

```diff
@@ -239,14 +239,14 @@ export class User extends Observable<SubscriberUser> {
     permissions: WalletPermission[]
     walletType: ExternalWalletType
   }) {
+    // get latest user info
+    await this.getData()
```

Comment:
This change fixes the flow where user (in oreid-portal) tried to connect their wallet and it threw error `Error: Public Key of signed string not found in wallet` - Tested for `ethereum` chain

**Flow**
- In oreid-portal, Select a dApp from "My Dapps"
- Click on "Connect Wallet"
- Login to the dApp
- Connect the wallet & sign string
- FAILS THE FIRST TIME
- try again - this time it works

The root issue was it failed at condition `if (!this.accountName || isNullOrEmpty(permissions)` as `this.accountName` was undefined/null - calling `await this.getData()` prior to this condition fixes the issue